lib/Mojolicious/Guides/Rendering.pod

```diff
@@ -819,8 +819,8 @@ logic with methods like L<Mojolicious::Validator::Validation/"is_valid">.
     my $validation = $c->validation;
     return $c->render unless $validation->has_data;
 
-    # Validate parameters ("pass_again" depends on "pass")
-    $validation->required('user')->size(1, 20)->like(qr/^[e-t]+$/);
+    # Validate parameters ("user" size is withing 1 - 20 and content looks somewhat like a valid unix username)
```

Comment:
Changed the code and update the comment, silently ignoring that the comment was not altered after a copy/paste.